internal/daemonset/daemonset.go

```diff
@@ -195,6 +200,25 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(ctx context.Context, d
 		Selector: &metav1.LabelSelector{MatchLabels: standardLabels},
 	}
 
+	if fw := mod.Spec.ModuleLoader.Container.Modprobe.FirmwarePath; fw != "" {
+		firmwareVolume := v1.Volume{
+			Name: nodeVarLibFirmwareVolumeName,
+			VolumeSource: v1.VolumeSource{
+				HostPath: &v1.HostPathVolumeSource{
+					Path: nodeVarLibFirmwarePath,
```

Comment:
do we want to create a path per Module? something like: /var/lib/firnware/<module-name>?
This way we avoid collision between different firmwares with the same name, and also hide the directory from user

Reply:
If I understand [the docs](https://www.kernel.org/doc/html/v5.19/driver-api/firmware/fw_search_path.html) correctly, it's global.

Reply:
We have the same understanding. I tried to path a `fw_path` parameter to the module, but it's not recognized. Everytime we add a search path to the kernel, the node is rebooted, so it's probably not what we want.

Reply:
> Everytime we add a search path to the kernel, the node is rebooted, so it's probably not what we want.

I guess writing to `/sys/module/firmware_class/parameters/path` doesn't reboot the node though, does it?